mindone/models/dit.py

```diff
@@ -564,6 +564,7 @@ def construct(self, x: Tensor, t: Tensor, y: Tensor):
         x = self.unpatchify(x)  # (N, out_channels, H, W)
         return x
 
+    @ms.jit
```

Comment:
Pls do not add this decorator here. Because `cfg_scale` is a float number instead of a tensor value, it will cause repetitive graph compilation on Ascend devices, which leads to slow sampling.

Reply:
will not recompile if cfg_scale is not changing during the whole inference stage, so it should be fine.

Reply:
OK, I see